modules/appcommands/commonAppCommands.go

```diff
@@ -676,3 +689,66 @@ func (a *app) setupConfigApplyCommand() {
 
 	a.ConfigApplyCmd.ArgStore = make(map[string]interface{})
 }
+
+func (a *app) setupKeystoreCreateCommand() {
+	a.KeystoreCreateCmd.Cmd = &cobra.Command{
+		Use:   a.KeystoreCreateCmd.Use,
+		Short: a.KeystoreCreateCmd.DescShort,
+		Long:  a.KeystoreCreateCmd.DescLong,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			additionalTest := a.KeystoreCreateCmd.AdditionalPreRunTest
+			err := a.setupDefaultConfigIfNotExists()
+			if err != nil {
+				return err
+			} else if err == nil && additionalTest != nil {
+				return additionalTest(cmd, args)
+			}
+			return nil
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			keystorePassPath := a.KeystoreCreateCmd.getStringFromArgStoreOrDie("pass-path")
+			home, err := util.GetUser()
+			if err == nil {
+				keystoreDir := home.HomeDir + "/.marlin/ctl/keystore/" + a.ProjectID
```

Comment:
we already have `/common` for each project in form of `/.marlin/ctl/a.ProjectID/common/`. This is used already in iris. Can follow that convention for common element too.

Reply:
done